### ISSUE TITLE
MariaDB - Table B stops working when table A triggers an INSERT on B

### ISSUE THREAD

**Issue by studiostart** [catfan/Medoo#1112]
Hello,
I'm using Medoo to work with my MariaDB database. It's fantastic, but I don't understand why a strange issue has happened.

I have two simple tables with just an ID and a string column. I'm going to call them A and B.

On both, I can execute SELECT and INSERT queries without problems. However, recently I introduced a Trigger on A, which populates B with just two rows (executing two simple INSERT queries). This Trigger seems to block table B, so it suddenly became unreadable.

Basically, this is what happens:

I execute "SELECT *" on B, and it works
I execute "INSERT" on A, and it works.
A runs the Trigger: two "INSERT" from this Trigger add two new rows on B
I execute "SELECT *" on B, and it fails
From this moment, I can't read table B and all the SELECT queries always return **False** (not an array as I expect)

Does anyone have experience with this type of behaviour?
